### DISCUSSION TITLE
adding a sound notification for 20% - 15% - 10% - 5% wireless headset battery status (Linux)

### DISCUSSION THREAD

**Discussion by y0ur5h4d0w** [Sapd/HeadsetControl#241] [Ideas]
hi all! 
is there a way to add a sound notification for when the headset is at a set battery percentage? i'm using a Corsair Void Elite RGB wireless headset and would be amazing to have a sound playing when the battery is low, the original iCue has a voice that tells the battery is low at around 15%, the ability to set a personalized sound for a personalized battery percentage would be amazing, sadly i can't code and i can't implement it myself :\ i'm still a noob in linux and programming

**Comment by Nama** [comment]
You can send a desktop notification with `notify-send`.
Just autostart this. But try it in the terminal first.

```sh
#!/bin/bash
while true; do
    batt=$(headsetcontrol -b -c)
    if (( "$batt" <= 10 )); then
        notify-send "Headset battery" "$batt"
        headsetcontrol -n 0
        headsetcontrol -n 1
    fi
    sleep 60
done
```
You can lookup a command to play an audio file, if the headset doesn't make any sound with `headsetcontrol -n 0/1`.
I didn't fully test this.
